templates/section.html.slim

```diff
@@ -63,7 +63,13 @@
     data-background-video-muted=((attr? 'background-video-muted') || (option? 'muted'))
     data-background-opacity=(attr "background-opacity")
     data-autoslide=(attr "autoslide")
-    data-state=(attr 'state'))
+    data-state=(attr 'state')
+    data-auto-animate=((attr? 'auto-animate') || (option? 'auto-animate'))
```

Comment:
We need to add a sample presentation in the `examples` directory to demonstrate how to use these attributes/options.

It serves multiple purposes:
- we can manually test that everything is working fine by opening the generated presentation in a browser
- we can write tests (using the existing test infrastructure) 
- we are "self-documenting" new features


I think we should also add a new page in the documentation named `auto-animate.adoc` to describe this new feature in the context of an AsciiDoc presentation.